templates/full-time-program.html

```diff
@@ -267,9 +270,10 @@ <h1 class="blue-h1">Application Process</h1>
       For the January to June ("H1") cohort, the application forms open, emails to
       interested folks go out, and forms close again around September.
       For the July to December ("H2") cohort, the application forms open,
-      emails to interested folks go out, and forms close again around March.
-      The application process continues for about five weeks after applications
-      close, with candidates eliminated at each step of the process.
+      emails to interested folks go out, and forms close again around March. 
+      The July 2024 cohort interest emails will go out in April and the application 
```

Comment:
For the 2024 H2 cohort, applications open in April and close in May.

Reply:
### ✅ Addressed in [476c912](https://github.com/Techtonica/techtonica.org/pull/336/commits/476c912449d5aed0a1343b11235d47ed52ed07db).